.github/workflows/backend-testing.yml

```diff
@@ -56,4 +56,4 @@ jobs:
       - name: Build debezium-checkstyle
         run: mvn clean install -f debezium/pom.xml -pl support/checkstyle -DskipTests -DskipITs
       - name: Test Debezium UI Backend
-        run: mvn clean install -f dbzui -B -Ddebezium.test.records.waittime=20 -Dformat.formatter.goal=validate -Dformat.imports.goal=check -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+        run: mvn clean install -f dbzui -B -Ddebezium.test.records.waittime=40 -Dformat.formatter.goal=validate -Dformat.imports.goal=check -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```

Comment:
This can be reverted. Higher timeouts will lead to builds stopping during build.

Reply:
Yeah, I will revert.  It was my attempt to get the backend test to complete.